graph.py

```diff
@@ -0,0 +1,77 @@
+
+class SimpleGraph(object):
+    '''This is a simple graph program that will allow us
+                to impliment a graph data structure'''
+    def __init__(self, dict_graph={}):
+        self.dict_graph = dict_graph
```

Comment:
This is a HUGE problem.  Create yourself a number of graphs and watch what happens.